src/Compilers/CSharp/Test/Semantic/Semantics/AmbiguousOverrideTests.cs

```diff
@@ -351,9 +351,9 @@ class CBar : IFoo // CS0535 * 2
     // (7,14): error CS0535: 'CBar' does not implement interface member 'IFoo.M<T>(T)'
     // class CBar : IFoo // CS0535 * 2
     Diagnostic(ErrorCode.ERR_UnimplementedInterfaceMember, "IFoo").WithArguments("CBar", "Metadata.IFoo.M<T>(T)").WithLocation(7, 14),
-    // (2,21): error CS0570: 'IFooAmbiguous<T, R>.M(?)' is not supported by the language
```

Comment:
>? [](start = 52, length = 1)

nit: I like that we load the type even if there is a modifier problem. But it feels like the change in diagnostics is making troubleshooting worse, as the `?` would at least point users closer to the source of the problem. We can tweak if we get feedback...

Reply:
> it feels like the change in diagnostics is making troubleshooting worse, as the ? would at least point users closer to the source of the problem. We can tweak if we get feedback...

I don't think a "?" actually points to any specific source of the problem

---
In reply to: [441162316](https://github.com/dotnet/roslyn/pull/45220#discussion_r441162316) [](ancestors = 441162316)

Reply:
I think The error is actually more informative now because you can use the info to locate the member in metadata. The signature is not mangled

---
In reply to: [441163418](https://github.com/dotnet/roslyn/pull/45220#discussion_r441163418) [](ancestors = 441163418,441162316)